recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "autogluon.multimodal" %}
-{% set version = "1.1.1" %}
```

Comment:
Do we miss this in the 1.2 release, or is it carried over from the last patch release for 1.1.1? If it’s the latter, let’s ensure the master branch is always up to date with the latest version, and we can merge version-specific updates into the versioned branch after the release.

Reply:
Yes it was carried over from the last release. Good suggestion to merge to the versioned branch.